apheleia-formatters.el

```diff
@@ -158,7 +158,7 @@
     (python3-json
      . ("python3" "-m" "json.tool"
         (apheleia-formatters-indent "--tab" "--indent")))
-    (rubocop . ("rubocop" "--stdin" filepath "--auto-correct"
+    (rubocop . ("rubocop" "--stdin" filepath "--autocorrect"
```

Comment:
This will break old or maintenance-mode projects using prior versions of rubocop; can we just use the "-a" alias?